go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/coreos/go-oidc/v3
+
+go 1.14
+
+require (
+	github.com/coreos/go-oidc v2.2.1+incompatible
```

Comment:
One suggestion is to run:

```
go mod tidy && go mod verify
```

Which will remove v2 references in go.mod and go.sum

Reply:
woops, yep done